crates/bevy_window/src/window.rs

```diff
@@ -625,6 +639,8 @@ pub struct WindowDescriptor {
     /// macOS X transparent works with winit out of the box, so this issue might be related to: <https://github.com/gfx-rs/wgpu/issues/687>
     /// Windows 11 is related to <https://github.com/rust-windowing/winit/issues/2082>
     pub transparent: bool,
+    /// A string that represents a DOM query selector for the target canvas element.
+    /// Use this in case you want to create the canvas element yourself.
     #[cfg(target_arch = "wasm32")]
     pub canvas: Option<String>,
```

Comment:
This field is changed a little and modified in https://github.com/bevyengine/bevy/pull/4726

Reply:
I will check it out.

Reply:
Looking at the change there's nothing that doesn't reflect the current documentation that I am adding.